Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -431,26 +440,50 @@ open class ComposerVC: _ViewController,
         present(filePickerVC, animated: true)
     }
     
-    /// Returns actions for attachments picker.
-    open var attachmentsPickerActions: [UIAlertAction] {
```

Comment:
getting rid of this is a breaking change, IMHO we should keep it and play along (add a `showCameraAction` in this variable)

Reply:
Yep good points, resolved now.